services/autoscaling/tests/unit/test_modules_auto_scaling_dynamic.py

```diff
@@ -2061,3 +2073,268 @@ async def test_warm_buffers_are_started_to_replace_missing_hot_buffers(
         len(analyzed_cluster.pending_ec2s)
         == app_settings.AUTOSCALING_EC2_INSTANCES.EC2_INSTANCES_MACHINES_BUFFER
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
Q: why not add on a single parametrize?

```python
@pytest.mark.parametrize("with_docker_join_drained, with_drain_nodes_labelled", ...)
async def test_(... )
   ...
```

Reply:
no reason. I did not check that one could put them together.